napari-workshops/notebook_setup.md

```diff
@@ -38,44 +39,58 @@ this tutorial.
  git clone https://github.com/napari/napari-workshop-template.git
  ```
 
-## Launch the `jupyter notebook` application
+## Install the additional `jupytext` dependency
 
-Open your terminal and navigate to the `notebooks` subdirectory of the
-`napari-workshop-template` directory you just downloaded.
+Because the notebooks were converted to MyST Markdown files (with a .md extension),
+to better visualize on GitHub and provide a nice rendered look on the web,
+you need to install one additional dependency to run them locally.
+
+````{important}
+Remember to activate the right environment!
+
+```bash
+conda activate napari-workshop
+```
+````
+
+```bash
+conda install jupytext jupyterlab-myst
```

Comment:
I've never had to install jupyterlab-myst, can you tell me what this does differently from whatever comes with jupytext?

Reply:
It makes notebooks in jupyter have the markdown admonitions rendered nicely, see the screenshot here:
https://github.com/napari/napari-workshop-template/pull/29

Reply:
Ooooh sounds great! I think I remember us discussing that at some point. Thanks!